arcgis-ios-sdk-samples/Info.plist

```diff
@@ -39,56 +39,63 @@
 	<dict>
 		<key>NSExceptionDomains</key>
 		<dict>
-			<key>buienradar.nl</key>
+			<key>arcgis.com</key>
```

Comment:
The exceptions for `arcgis.com` and `arcgisonline.com` pre-existed my changes. I alphabetized the exceptions so that's why they appear changed. We can try removing them in a separate PR.